definitions/infra-kubernetes_daemonset/definition.yml

```diff
@@ -28,4 +28,6 @@ synthesis:
     newrelic.agentVersion:
         multiValue: false
     aws.region:
+      multiValue: false
     aws.ec2.instanceType:
+      multiValue: false
```

Comment:
same question than in another pr. why do we think these ones will not be multi value?

Reply:
We decided to actually drop this tag. But I'll explain it below.

I leave to think in reverse: what would be the implications of this tag being multi-value? This can only be answered by knowing the source of this tag.

This tag comes from data the agent fetches about the machines where it runs, when these machines are in AWS. They are not related to any monitored daemonset.

Then we get back to implications: you would see a daemonset instance with many different "instanceTypes", with values changing as the integration's pod get moved between nodes. This is extremely confusing and not helpful at all. So we don't want "many historical values", we want only the most recent.

Reply:
Thanks! That makes a lot of sense